tests/jobs/overburden_timeshift/test_ots_config.py

```diff
@@ -7,7 +7,8 @@
 from datetime import datetime
 
 
-def test_valid_config(tmpdir, monkeypatch):
+@pytest.mark.parametrize("velocity_model", [True, False])
```

Comment:
You do like parameters :)

Reply:
In this case I think they improve readability, but I can easily see cases where they do the opposite.

Reply:
But it won't be right? 
```python
if velocity_model is not None:
          conf.update({"velocity_model": velocity_model})
```

Just thinking how it writes (if test fails `velocity_model` True / False instead of `norne_vol.segy, None` which in 1 year time would give more sense to me when reading it :) But I left it up to your then.

Reply:
I am a bit confused? My suggestion is equal to yours, right? Do you not want to update the `dict`?

Reply:
Oh, haven't realized it was a question.